generatep

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Runs go generate for each directory, but in parallel. Any arguments are appended to the
+# go generate command.
+# Usage: $ ./generatep [go generate arguments]
+# Print all generate commands: $ ./generatep -x
+
+cd ./wayland
+find . -type f -name '*.go' -exec dirname {} \; | sort -u | parallel -j 0 go generate $1 {}/.
```

Comment:
How this works:
1. Get all go files and get their directories.
2. Filter out duplicates.
3. Run `go generate $customArgs $dir/.` for each dir, in parallel, with unlimited jobs.

With `./generatep -x`, this results in the following command `go generate -x ./unstable/xdg-output-v1/.`